root/src/scss/base.scss

```diff
@@ -14,4 +14,9 @@
 .root-loading {
   @extend %vf-has-box-shadow;
   background-color: $color-x-light;
+  padding-top: $spv-inner--medium;
+
+  h4 {
```

Comment:
Normally I would suggest not to apply styles on element selectors (i.e. the `h4`) but instead give it a class (`root-loading__title`?) in order to better clarify what exactly is being styled as well as preventing "leaky" styling. But since this whole workspace is gonna be removed pretty soon it's no big deal.